src/adapters/node.ts

```diff
@@ -1,5 +1,4 @@
 import type { AdapterOptions, AdapterInstance, Adapter } from "../adapter.ts";
-import type { WebSocket } from "../../types/web.ts";
```

Comment:
This import wasn't being used. There's another similar import on a different line but uses an import alias `WebSocket as WebSocketT,`